.luacheckrc

```diff
@@ -1,4 +1,4 @@
-globals = {"todo", "mod_gui", "script", "game", "defines", "data", "serpent", "settings", "global"}
+globals = {"todo", "mod_gui", "script", "game", "defines", "data", "serpent", "settings", "global", "storage"}
```

Comment:
It should rather replace "global", right?

```suggestion
globals = {"todo", "mod_gui", "script", "game", "defines", "data", "serpent", "settings", "storage"}
```

Reply:
Yep fair point